Xamarin.Forms.Build.Tasks/Xamarin.Forms.Build.Tasks.csproj

```diff
@@ -29,11 +29,33 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 
+  <PropertyGroup>
+     <UpdateXlfOnBuild Condition="'$(CI)' != 'true'">true</UpdateXlfOnBuild>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(CI)' != 'true'">
+     <PackageReference Include="XliffTasks" Version="1.0.0-beta.20206.1" />
```

Comment:
Should this have `PrivateAssets="all"` ?

Reply:
forgot to commit this. fixed